galaxy_ng/app/api/v1/tasks.py

```diff
@@ -286,18 +289,33 @@ def legacy_role_import(
     there will not be any duplicate key errors. However,
     the "commit" field should always reflect the latest import.
     """
-    logger.info('START LEGACY ROLE IMPORT')
-    logger.info(f'REQUEST_USERNAME:{request_username}')
-    logger.info(f'GITHUB_USER:{github_user}')
-    logger.info(f'GITHUB_REPO:{github_repo}')
-    logger.info(f'GITHUB_REFERENCE:{github_reference}')
-    logger.info(f'ALTERNATE_ROLE_NAME:{alternate_role_name}')
-    logger.info(f'superuser_can_create_namespaces: {superuser_can_create_namespaces}')
 
-    clone_url = None
+    task = None
+    try:
+        task = Task.current()
+    except Exception:
+        pass
+    if task:
+        task_id = task.pulp_id
+        LegacyRoleImport.objects.get_or_create(task_id=Task.current().pulp_id)
 
+    logger.info('starting legacy role import')
```

Comment:
Can we drop 'legacy' here? Or at least replace it with 'standalone'?
```suggestion
    logger.info('starting role import')
```